pkg/parser/parser.go

```diff
@@ -36,6 +46,25 @@ func HTML(document string) string {
 		extensions = append(extensions, &wikilink.Extender{})
 	}
 
+	if EnableDefinitionList {
```

Comment:
This section needs a rewrite. I wasn't sure if it was a good idea with optional extensions or not when adding wikilinks, I am still not sure if I am entirely convinced, but with this many at least the code needs an improvement.

Consider something like this:
```
	optionalExtensions := map[goldmark.Extender]bool{
		&wikilink.Extender{}:     EnableWikiLinks,
		extension.DefinitionList: EnableDefinitionList,
		extension.Footnote:       EnableFootnotes,
		extension.Typographer:    EnableTypographer,
		extension.CJK:            EnableCJK,
		emoji.Emoji:              EnableEmoji,
		figure.Figure:            EnableCaptions,
	}

	for ext, enabled := range optionalExtensions {
		if enabled {
			extensions = append(extensions, ext)
		}
	}
```